### PR TITLE
Minor Mobile Improvements

### PR DESCRIPTION
**Pull request by pyxld-kris**
This PR includes a bunch of small mobile touchups, as per issue #753

For quick reference:
```
Consideration:
We need to fix the top navigation and the home page for mobile and also the sign in button in the workshopping page isn't visible.

Acceptance criteria:

Top navigation and home page works well for mobile
Sign in button in the workshopping page is visible and works well for web responsive.
```

Before creating this PR, I merged `development/ideaspace` into this feature branch. This should hopefully save you some work Dave!

NOTE: There's an issue with the header in master currently which pushes the page width to be wider than it should. That will make the mobile stuff look less correct, but fixing that is outside the scope of this PR.
image

### CODE REVIEW

File: apps/ideaspace/src/components/common/SignInButton/StyledSignInButton.js [hunk 1]
```diff
@@ -3,6 +3,6 @@ import Button from "../Button";
 
 export const MenuButton = styled(Button)`
   @media (orientation: portrait) {
-    display: none;
+    //display: none;
```
Comment (dbradham) [inline apps/ideaspace/src/components/common/SignInButton/StyledSignInButton.js:6]: Is this comment necessary? Could we delete this line altogether?

File: apps/ideaspace/src/components/modules/WelcomePage/StyledWelcomePage.js [hunk 2]
```diff
@@ -3,9 +3,10 @@ import Link from 'next/link';
 
 export const ButtonArea = styled.div`
   width: 100%;
-  height: 50vh;
+  min-height: 50vh;
+  height: 1px;
   max-width: 1700px;
-  max-height: 500px;
+  //max-height: 500px;
```
Comment (dbradham) [inline apps/ideaspace/src/components/modules/WelcomePage/StyledWelcomePage.js:9]: instead of commenting, could we just delete this line?

**Comment by pyxld-kris** [comment]
Fixes have been applied, thanks Dave!

**Review by dbradham — APPROVED** [review]

**Pull request merged**